File: Projeto_5/src/funcionario_administrativo.py
from usuario import Usuario

# criaçao da class FuncionarioAdministrativo
class  FuncionarioAdministrativo(Usuario):
    def __init__(self, nome: str, email: str, departamento: str, cargo: str):
        super().__init__(nome, email, "Funcionário Administrativo")
        self.departamento = departamento
        self.cargo = cargo

    def gerarRelatorio(self):
        return f"Funcionário: {self.nome}, Departamento: {self.departamento}, Cargo: {self.cargo}"

File: Projeto_5/funcionario.administrativo.cpp
#include "funcionario.administrativo.h"
#include <iostream>

FuncionarioAdministrativo::FuncionarioAdministrativo() : Usuario(), departamento(""), cargo("") {}

FuncionarioAdministrativo::FuncionarioAdministrativo(const std::string& nome, const std::string& email, const std::string& departamento, const std::string& cargo)
    : Usuario(nome, email, "Funcionário Administrativo"), departamento(departamento), cargo(cargo) {}

void FuncionarioAdministrativo::gerarRelatorio() const {
    std::cout << "Relatório do Funcionário Administrativo\n";
    std::cout << "Nome: " << nome << "\nEmail: " << email << "\nDepartamento: " << departamento << "\nCargo: " << cargo << "\n";
}
